std/stdio.d

```diff
@@ -4522,70 +4521,66 @@ Initialize with a message and an error code.
     }
 }
 
-extern(C) void std_stdio_static_this()
+// Undocummented but public because std* handles are aliasing it
+ref File makeGlobal(alias handle)()
 {
-    static import core.stdc.stdio;
-    //Bind stdin, stdout, stderr
-    __gshared File.Impl stdinImpl;
-    stdinImpl.handle = core.stdc.stdio.stdin;
-    .stdin._p = &stdinImpl;
-    // stdout
-    __gshared File.Impl stdoutImpl;
-    stdoutImpl.handle = core.stdc.stdio.stdout;
-    .stdout._p = &stdoutImpl;
-    // stderr
-    __gshared File.Impl stderrImpl;
-    stderrImpl.handle = core.stdc.stdio.stderr;
-    .stderr._p = &stderrImpl;
+    static __gshared File.Impl impl;
+    static __gshared File result;
+    static shared bool lilyWasHere;
```

Comment:
Ahem... not that it's bad to have fun, but I don't think such names should be present in a **standard** library. We already have a few words in there like "Stuff", that are really non-descript. Or messages worded in terms of "Dunno how to do <stuff>", which someone will eventually find difficult to understand. This one is completely internal and not supposed to be looked at all that much, but a person who eventually **does** have to look is also most likely to go "What???"